lib/linter/linter.js

```diff
@@ -914,42 +914,7 @@ function createRuleListeners(rule, ruleContext) {
     }
 }
 
-// methods that exist on SourceCode object
-const DEPRECATED_SOURCECODE_PASSTHROUGHS = {
-    getSource: "getText",
-    getSourceLines: "getLines",
-    getAllComments: "getAllComments",
-    getNodeByRangeIndex: "getNodeByRangeIndex",
-    getComments: "getComments",
-    getCommentsBefore: "getCommentsBefore",
-    getCommentsAfter: "getCommentsAfter",
-    getCommentsInside: "getCommentsInside",
-    getJSDocComment: "getJSDocComment",
-    getFirstToken: "getFirstToken",
-    getFirstTokens: "getFirstTokens",
-    getLastToken: "getLastToken",
-    getLastTokens: "getLastTokens",
-    getTokenAfter: "getTokenAfter",
-    getTokenBefore: "getTokenBefore",
-    getTokenByRangeStart: "getTokenByRangeStart",
-    getTokens: "getTokens",
-    getTokensAfter: "getTokensAfter",
-    getTokensBefore: "getTokensBefore",
-    getTokensBetween: "getTokensBetween"
-};
-
-
-const BASE_TRAVERSAL_CONTEXT = Object.freeze(
-    Object.keys(DEPRECATED_SOURCECODE_PASSTHROUGHS).reduce(
-        (contextInfo, methodName) =>
-            Object.assign(contextInfo, {
-                [methodName](...args) {
-                    return this.sourceCode[DEPRECATED_SOURCECODE_PASSTHROUGHS[methodName]](...args);
-                }
-            }),
-        {}
-    )
-);
+const BASE_TRAVERSAL_CONTEXT = {};
```

Comment:
Do we still need `BASE_TRAVERSAL_CONTEXT`? Looks like we can just remove it now.

Reply:
Ah, right you are!